dali/operators/segmentation/random_object_bbox.cc

```diff
@@ -436,12 +459,31 @@ bool RandomObjectBBox::PickForegroundBox(
   InitClassInfo(context.sample_idx);
   context.class_label = class_info_.background;
   auto &tp = *context.thread_pool;
+
+  CacheEntry *cache_entry = nullptr;
+  kernels::fast_hash_t hash = {};
+  if (use_cache_) {
+    fast_hash(hash, input.data, input.num_elements() * sizeof(T));
```

Comment:
FYI: This hash is so fast, that running it in parallel resulted in overall slowdown.

Reply:
So it is memory bound?

Reply:
Yes, it's 2x faster than memcpy on my machine.